examples/cifar/train.py

```diff
@@ -56,6 +58,18 @@ def run(args):
                             dropout_rate=do, num_classes=num_classes)
     elif args.model.lower() == 'mobilenet':
         model = MobileNet(num_classes=num_classes)
+    elif args.model.lower() == 'vit':
+        model = ViT(
+            image_size = 32,
+            patch_size = 4,
+            num_classes = 10,
```

Comment:
use the `num_classes` variable that is defined above, this way it will also work with cifar100.

Reply:
make sense, I'll update.